react/Checkbox/Checkbox.js

```diff
@@ -110,19 +112,19 @@ export default class Checkbox extends Component {
   }
 
   render() {
-    const { className, id, valid, message, messageProps } = this.props;
+    const { className, id, valid, message, messageProps, tone } = this.props;
 
     const rootClassNames = classnames({
       [styles.root]: true,
-      [styles.invalid]: valid === false,
+      [styles.invalid]: typeof tone !== 'undefined' ? tone === TONE.CRITICAL : valid === false,
```

Comment:
Probably could specifically check for oneOf (in values) the types, otherwise anything passed will break it

Reply:
Yes there's a prop type check for `tone` in this component so console warnings will show up if an invalid value is passed in